CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [3.0.3] - 2017-09-21
```

Comment:
As there is no guarantee on when things get merged it is best to leave the versioning and date to maintainer and put all changes under `## [Unreleased]` [per](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change).